src/randoop/main/GenInputsAbstract.java

```diff
@@ -409,17 +373,12 @@ public GenInputsAbstract(String command, String pitch,
   public static boolean dont_output_tests = false;
   
   /**
-   * Output sequences even if they do not complete execution.
-   * 
-   *  Randoop's default behavior is to output only tests consisting of
-   *  method call sequences that execute every statement, rather than throwing
-   *  an exception or failing a contract check before the last statement.
+   * Indicate which classes that any generated test must include. Tests that
+   * do not have classes matching the regular expression are not written to 
+   * the test suites. 
    */
-  @Option("Output sequences even if they do not complete execution")
-  public static boolean output_nonexec = false;
-  
-  @Option("specifies regex of classes that must be in any regression tests")
-  public static Pattern test_classes = null;
+  @Option("Regular expression for class names that must occur in generated tests")
```

Comment:
I already changed it so that it checks the declaring class of the operation
in each statement instead of matching against the variable names.

On Mon, Dec 28, 2015 at 1:30 PM, Michael Ernst <notifications@github.com>
wrote:

> In src/randoop/main/GenInputsAbstract.java
> <https://github.com/randoop/randoop/pull/73#discussion_r48506890>:
>
> >     */
> > -  @Option("Output sequences even if they do not complete execution")
> > -  public static boolean output_nonexec = false;
> > -
> > -  @Option("specifies regex of classes that must be in any regression tests")
> > -  public static Pattern test_classes = null;
> > +  @Option("Regular expression for class names that must occur in generated tests")
>
> That concern sounds right. You could add that as low-priority item to your
> to-do list.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/randoop/randoop/pull/73/files#r48506890>.
>



-- 
Ben Keller
bjkeller@uw.edu